instrumentation/network/src/main/java/io/opentelemetry/android/instrumentation/network/NetworkChangeMonitor.java

```diff
@@ -20,33 +20,26 @@
  */
 public final class NetworkChangeMonitor {
 
-    public static NetworkChangeMonitor create(CurrentNetworkProvider currentNetworkProvider) {
-        return builder(currentNetworkProvider).build();
-    }
-
-    public static NetworkChangeMonitorBuilder builder(
-            CurrentNetworkProvider currentNetworkProvider) {
-        return new NetworkChangeMonitorBuilder(currentNetworkProvider);
+    public NetworkChangeMonitor(
+            CurrentNetworkProvider currentNetworkProvider,
+            List<AttributesExtractor<CurrentNetwork, Void>> additionalExtractors) {
+        this.currentNetworkProvider = currentNetworkProvider;
+        this.additionalExtractors = additionalExtractors;
     }
 
     private final CurrentNetworkProvider currentNetworkProvider;
     private final List<AttributesExtractor<CurrentNetwork, Void>> additionalExtractors;
 
-    NetworkChangeMonitor(NetworkChangeMonitorBuilder builder) {
-        this.currentNetworkProvider = builder.currentNetworkProvider;
-        this.additionalExtractors = new ArrayList<>(builder.additionalExtractors);
-    }
-
     /**
-     * Installs the network change monitoring instrumentation on the given {@link
-     * InstrumentedApplication}.
+     * Installs the network change monitoring instrumentation on the given {@link OpenTelemetry}.
      */
-    public void installOn(InstrumentedApplication instrumentedApplication) {
+    public void installOn(OpenTelemetry openTelemetry) {
```

Comment:
This no longer makes sense as it is written. It used to be "Hey network monitor, install yourself into this application", but it now reads "Hey install yourself into this OpenTelemetry" which I think doesn't make sense. It's also odd that the otel instance isn't just passed to the constructor, since the caller has it at creation time.

We can defer this naming choice until part 3 I suppose, but maybe we should remove the otel param, pass it in the constructor instead, and just call this method `start()`.